### ISSUE TITLE
Medication Log 

### ISSUE THREAD

**Issue by alex-reardon** [alex-reardon/PPMI_merge_Readme#12]
https://github.com/alex-reardon/PPMI_merge_Readme/blob/b163bf4060ad474f3a9dff35fc4ce262fda55714/ppmi_merge.Rmd#L239

- Not a useful column

**Comment by pvaswani** [comment]
Agreed, as shown. Providing subject non-PD medications in some way would be valuable, for example if a researcher wanted to look at treatment of nonmotor symptoms. Not sure how best to do so given the plethora of medications, with misspellings (see Amlodipine and 'amlodipin' in the example), and also providing treatment dates.
